Doc/whatsnew/3.11.rst

```diff
@@ -243,8 +243,8 @@ time
 ----
 
 * On Unix, :func:`time.sleep` now uses the ``clock_nanosleep()`` or
-  ``nanosleep()`` function, if available, which has a resolution of 1 ns (10^-6
-  sec), rather than using ``select()`` which has a resolution of 1 us (10^-9
+  ``nanosleep()`` function, if available, which has a resolution of 1 ns (10^-9
+  sec), rather than using ``select()`` which has a resolution of 1 us (10^-6
```

Comment:
```suggestion
  ``nanosleep()`` function, if available, which has a resolution of 1 ns (10\ :sup:`-9`
  sec), rather than using ``select()`` which has a resolution of 1 us (10\ :sup:`-6`
```